Markdown/Markdown.sublime-syntax

```diff
@@ -898,18 +915,285 @@ contexts:
         - match: $
           pop: true
   fenced-code-block:
+    - match: |-
+         (?x)
+          {{fenced_code_block_start}}
+          ((?i:xml))
+          \s*$          # any whitespace until EOL
+      captures:
+        2: markup.raw.code-fence.xml.markdown-gfm punctuation.definition.raw.code-fence.begin.markdown
```

Comment:
I don't think these captures should be getting the `markup.raw` scope. They aren't part of the "raw" segment since they are not rendered verbatim. (Also, I have dimmed background for `markup.raw` and it would be nice if that started on the first line of the raw block.)

Reply:
And the `escape_captures` for `0` should be removed.